contrib/init/datum_gateway.service

```diff
@@ -0,0 +1,89 @@
+[Unit]
+Description=DATUM Gateway
+Documentation=https://github.com/OCEAN-xyz/datum_gateway
+
+#
+# Expected file locations when using hardening options:
+# binary: /usr/bin/datum/
+# configuration: /etc/datum/
+# logs: /var/lib/datum_gateway/
+#
+
+# https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/
+Wants=network-online.target
+After=network-online.target
+# alternatively, wait until bitcoind is up
+#After=bitcoind.service
+
+[Service]
+
+ExecStart=/usr/bin/datum/datum_gateway --config=/etc/datum/datum_gateway_config.json
+
+# Make sure the config directory is readable by the service user
+PermissionsStartOnly=true
+ExecStartPre=/bin/chgrp datum /etc/datum
+
+#
+# Process Management
+#
+
+Type=simple
+NotifyAccess=all
+PIDFile=/run/datum/datum_gateway.pid
+
+Restart=on-failure
+TimeoutStartSec=infinity
+TimeoutStopSec=600
+
+# limit number of open file descriptors
+LimitNOFILE=65535
+
+#
+# Directory Creation & Permissions
+#
+
+User=datum
+Group=datum
+
+# /run/datum_gateway
+RuntimeDirectory=datum_gateway
```

Comment:
This doesn't match PIDFile above

Reply:
Typo in the path for PIDFile. Would fix, but… Actually, PIDFile isn't even needed when running with `Type=simple`. Will delete this instead.